Source/WebCore/dom/Node.h

```diff
@@ -710,6 +710,7 @@ class Node : public EventTarget {
     void updateAncestorsForStyleRecalc();
     void markAncestorsForInvalidatedStyle();
 
+    Vector<Ref<Node>, 1> convertNodesOrStringsIntoNodeList(FixedVector<NodeOrString>&&);
```

Comment:
I would suggest using NodeVector instead of NodeList. NodeList is a very specific DOM type and I thought this was what was being returned at first.

Reply:
Okay, I did debate between naming that vs. this so I'd change it.